contrib/aws/aws-sdk-go-v2/aws/aws.go

```diff
@@ -101,6 +126,228 @@ func (mw *traceMiddleware) startTraceMiddleware(stack *middleware.Stack) error {
 	}), middleware.After)
 }
 
+func extractResourceNameFromParams(requestInput middleware.InitializeInput, awsService string) (string, string, error) {
```

Comment:
```suggestion
func resourceNameFromParams(in middleware.InitializeInput, service string) (string, string, error) {
```
Generally reducing verbosity